website/src/views/helpers.ts

```diff
@@ -114,6 +115,15 @@ export type LapisMutationQuery = {
     aminoAcidInsertions?: string[];
 };
 
+export function getMutationFilter(mutationFilter: LapisMutationQuery): MutationFilter {
```

Comment:
This is essentially a duplicate of `getLapisMutations`, but with a better return type. We don't need both functions, do we?